sdv/tabular/base.py

```diff
@@ -439,7 +441,11 @@ def sample(self, num_rows=None, max_retries=100, max_rows_multiplier=10,
             if column not in self._metadata.get_fields():
                 raise ValueError(f'Invalid column name `{column}`')
 
-        transformed_conditions = self._metadata.transform(conditions, on_missing_column='drop')
+        try:
+            transformed_conditions = self._metadata.transform(conditions, on_missing_column='drop')
+        except ConstraintsNotMetError:
+            raise ValueError("The given conditions are not able to be met.")
```

Comment:
I would not raise a new exception, since this will add a new layer to the traceback.
I would just updated the message and re-raise:
```
except ConstraintsNotMetError as cnme:
    cnme.message = 'Passed conditions are not valid for the given constraints'
    raise
```